packages/editor/src/components/post-author/index.js

```diff
@@ -1,49 +1,165 @@
 /**
  * WordPress dependencies
  */
-import { __ } from '@wordpress/i18n';
+import { sprintf, __, _n } from '@wordpress/i18n';
 import { withInstanceId, compose } from '@wordpress/compose';
 import { Component } from '@wordpress/element';
 import { withSelect, withDispatch } from '@wordpress/data';
+import apiFetch from '@wordpress/api-fetch';
 
 /**
  * Internal dependencies
  */
 import PostAuthorCheck from './check';
 
+/**
```

Comment:
These should be listed first of the dependencies blocks, in order of increasing locality.

https://github.com/WordPress/gutenberg/blob/master/docs/contributors/coding-guidelines.md#javascript

Reply:
Addressed in https://github.com/WordPress/gutenberg/pull/7385/commits/93b5396bd55dfd09e13832d394ba6899a2f00c24